testing/src/main/java/io/stargate/it/bridge/BridgeIntegrationTest.java

```diff
@@ -137,4 +145,14 @@ protected QueryOuterClass.QueryParameters.Builder queryParameters(
   protected QueryOuterClass.QueryParameters.Builder queryParameters(CqlIdentifier keyspace) {
     return queryParameters(keyspace, false);
   }
+
+  protected static String sourceApi() {
```

Comment:
Since these are static and not meant for overridability, could just inline `sourceApi` value and interceptor closure.